npm/private/lifecycle/lifecycle-hooks.js

```diff
@@ -167,7 +167,19 @@ async function copyRecursive(src, dest) {
     try {
         await main(process.argv.slice(2))
     } catch (e) {
-        console.error(e)
+        if (e.code === "ELIFECYCLE" && e.pkgid && e.stage && e.script) {
+            console.log("===============================================================")
```

Comment:
how come these go to stdout rather than stderr? is there some reasoning for that?

Reply:
oh I see, you commented earlier. Consider leaving a comment in the code so someone doesn't change it later.

Reply:
Added a comment.